pyproject.toml

```diff
@@ -41,6 +41,10 @@ dev = [
 litellm = [
     "litellm>=1.52",
 ]
+transformers = [
+    "transformers>=4.46.0",
+    "accelerate>=1.2.1"
+]
```

Comment:
awesome! Is there a specific reason we need accelerate? perhaps we can releax the constraints for transformers a bit too? Should we add an overview of supported pipes somewhere?

Reply:
device_map is set to "auto" to have `accelerate` compute the most optimized device_map automatically. let's remove device_map="auto", there's no need to add the accelerate dependency just for that! i will address that tomorrow